canopen_motor_node/src/control_node.cpp

```diff
@@ -83,8 +88,16 @@ class MotorChain : public RosChain{
     MotorChain(const ros::NodeHandle &nh, const ros::NodeHandle &nh_priv): RosChain(nh, nh_priv), motor_allocator_("canopen_402", "canopen::MotorBase::Allocator"){}
 
     virtual bool setup() {
+
+        if(nh_priv_.param("has_robot_description", true)){
```

Comment:
In `control_node.cpp` the URDF is used to check for correct joints names and to parse the limits.
I should use the searchParam code here instead of of using `nh_priv`.